packages/checkoutcom-integration/src/checkoutcomFieldsets/getCheckoutcomFieldsetValidationSchemas.tsx

```diff
@@ -56,6 +56,16 @@ const checkoutComShemas: {
     }),
 };
 
+export type TranslateValidationErrorFunction = (
```

Comment:
It might be better to move this type to payment-integration-api package, because it can be useful for other packages in the future as well.